src/store/edgeImages.js

```diff
@@ -29,6 +36,7 @@ export default applyReducerHash(
   {
     [`${LOAD_EDGE_IMAGES}_PENDING`]: loadEdgeImagesPending,
     [`${LOAD_EDGE_IMAGES}_FULFILLED`]: loadEdgeImagesFulfilled,
+    [`${LOAD_EDGE_IMAGES}_SORTED`]: loadEdgeImagesSorted,
```

Comment:
no need for sorted. it is just overload on the reducer. when we sort we fetch images with new search query.